homeassistant/components/unifi/device_tracker.py

```diff
@@ -100,13 +156,34 @@ class UnifiEntityTrackerDescriptionMixin(Generic[_HandlerT, _DataT]):
 
 @dataclass
 class UnifiTrackerEntityDescription(
-    UnifiEntityDescription[_HandlerT, _DataT],
-    UnifiEntityTrackerDescriptionMixin[_HandlerT, _DataT],
+    UnifiEntityDescription[HandlerT, DataT],
+    UnifiEntityTrackerDescriptionMixin[HandlerT, DataT],
 ):
     """Class describing UniFi device tracker entity."""
 
 
 ENTITY_DESCRIPTIONS: tuple[UnifiTrackerEntityDescription, ...] = (
+    UnifiTrackerEntityDescription[Clients, Client](
+        key="Client device scanner",
+        has_entity_name=True,
+        allowed_fn=async_client_allowed_fn,
+        api_handler_fn=lambda api: api.clients,
+        available_fn=lambda controller, obj_id: controller.available,
+        device_info_fn=lambda api, obj_id: None,
+        event_is_on=WIRED_CONNECTION + WIRELESS_CONNECTION,
+        event_to_subscribe=WIRED_CONNECTION
```

Comment:
Would black keep an explicit parenthesis around the terms making it easier to read with the terms indented?

Reply:
It seems so, fixed.